ext/RMagick/extconf.rb

```diff
@@ -94,27 +94,28 @@ def configure_compile_options
         check_multiple_imagemagick_versions
         check_partial_imagemagick_versions
 
+        original_ldflags = $LDFLAGS.dup
         libdir  = `pkg-config --libs-only-L #{$magick_package}`.chomp.sub('-L', '')
         ldflags = "#{ENV['LDFLAGS']} " + `pkg-config --libs #{$magick_package}`.chomp
         rpath   = libdir.empty? ? '' : "-Wl,-rpath,#{libdir}"
 
         # Save flags
-        $CPPFLAGS   = "#{ENV['CPPFLAGS']} " + `pkg-config --cflags #{$magick_package}`.chomp
-        $LOCAL_LIBS = "#{ENV['LIBS']} "     + `pkg-config --libs #{$magick_package}`.chomp
-        $LDFLAGS    = "#{ldflags} #{rpath}"
+        $CPPFLAGS   += " #{ENV['CPPFLAGS']} " + `pkg-config --cflags #{$magick_package}`.chomp
```

Comment:
I'm not sure it's good idea to include the environment variable in here and in lines around.
That seems uncommon at least.
But since it doesn't cause an immediate issue I kept them just in case.